packages/link-bins/test/index.ts

```diff
@@ -40,105 +38,92 @@ function getExpectedBins (bins: string[]) {
   return expectedBins.sort()
 }
 
-test('linkBins()', async (t) => {
+test('linkBins()', async () => {
   const binTarget = tempy.directory()
-  t.comment(`linking bins to ${binTarget}`)
-  const warn = sinon.spy()
+  const warn = jest.fn()
 
   await linkBins(path.join(simpleFixture, 'node_modules'), binTarget, { warn })
 
-  t.notOk(warn.called)
-  t.deepEqual(await fs.readdir(binTarget), getExpectedBins(['simple']))
+  expect(warn).not.toHaveBeenCalled()
+  expect(await fs.readdir(binTarget)).toEqual(getExpectedBins(['simple']))
```

Comment:
Attempted to use `.toStrictEqual` here. However, the array from `fs.readdir` and `['simple']` fails strict equality within Jest's `typeEquality`

https://github.com/facebook/jest/blob/6b3a848159f3bd5b51542fe0f185c6545e7f3670/packages/expect/src/utils.ts#L314-L320

The constructors don't match.
Did some digging and both constructors are `Array.isArray` and both toString as `function() { [native code] }`.

This difference is only true when running in Jest (node 12.18.4 on MacOS).